module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,21 @@
+#include "sort.hpp"
+#include <algorithm>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& list) {
+    list.sort();
+    std::deque<std::string> dequeSorted;
+
+    for (auto &el : list) {
```

Comment:
const 
```suggestion
    for (const auto &el : list) {
```

Reply:
As I remember we can get rid of for loop using 
`std::deque<std::string> dequeSorted{list.begin(), list.end()};`